Arista/Ticks/hostdown.tick

```diff
@@ -18,29 +19,32 @@ var hosts = batch
   @withoutUpdates()
     .period(MAX_NOUPDATE_PERIOD)
     .roundDuration(HOST_CHECK_PERIOD)
-  |eval(lambda: "noupdates", lambda: UPDATES_IN_MAX_NOUPDATE_PERIOD, lambda: UPDATES_IN_MAX_NOUPDATE_PERIOD)
-    .as('failed', 'missingUpdates', 'expectedUpdates')
-    .keep('missingUpdates', 'expectedUpdates')
-
-hosts
-  |influxDBOut()
-    .database('ServerStats')
-    .retentionPolicy('default')
-    .measurement('hostStatus')
 
+// raise critical alert for servers which haven't updated within
+// MAX_NOUPDATE_PERIOD
 hosts
-  |count('missingUpdates')
-  |httpOut('downservers')
   |alert()
-    .stateChangesOnly()
-    .crit(lambda: "count" > 0)
+    .StateChangesOnly()
+    .crit(lambda: "noupdates" == 2)
     .id('kapacitor/{{ .TaskName }}/{{ index .Tags "host" }}')
     .message('{{ .Level }} - host down alert for Host {{ index .Tags "host" }}')
     .details('''
       <pre>
-        Kapacitor task {{ .TaskName }} raised {{ .Level }} alert on {{ index .Tags "host" }} at {{ .Time }}
+        Kapacitor task {{ .TaskName }} raised {{ .Level }} alert for {{ index .Tags "host" }} at {{ .Time }}
```

Comment:
Can we add a more specific message? To indicate this a critical alert because the host is detected being down for last 1 hour?